kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/QuizOptionsDropdownMenu.vue

```diff
@@ -26,26 +26,30 @@
     name: 'QuizOptionsDropdownMenu',
     mixins: [coachStringsMixin, commonCoreStrings],
     props: {
-      draft: {
-        type: Boolean,
-        default: false,
+      exam: {
+        type: Object,
+        required: false,
+        default: null,
       },
     },
     computed: {
       options() {
-        return [
-          {
-            label: this.draft
-              ? this.coreString('editAction')
-              : this.coreString('editDetailsAction'),
-            value: 'EDIT_DETAILS',
-          },
+        const options = [
           {
             label: this.$tr('copyQuizAction'),
             value: 'COPY',
           },
           { label: this.coreString('deleteAction'), value: 'DELETE' },
         ];
+        if (!this.exam?.archive) {
```

Comment:
Is this just because the `archive` key would exist so we can use that instead of the `draft` prop? If so, how is that distinct from `this.exam?.draft` that we are using below for the label condition?

Reply:
Oh no, there are three different "states". exam in draft, exam "active"(once the exam is published) and exam closed. The  `this.exam?.archive` represents that the exam was closed. So when the exam is "not archived", it can be either in draft or not.